### PR TITLE
Fix Link and admin1 details

### PR DESCRIPTION
**Pull request by roshni73**
## Addresses
-   https://github.com/IFRCGo/alert-hub-web-app/issues/54
-   https://github.com/IFRCGo/alert-hub-web-app/issues/52

## Changes
-  Add Link in all the value which includes url
- Fix admin1 list in the Alert detail page
## This PR doesn't introduce:
- [x] typos
- [x] conflict markers
- [x] unwanted comments
- [x] temporary files, auto-generated files or secret keys
- [x] `console.log` meant for debugging
- [x] codegen errors

### CODE REVIEW

**Review by barshathakuri — APPROVED** [review]

File: src/views/AlertDetails/AlertInfo/index.tsx [hunk 1]
```diff
@@ -232,7 +233,15 @@ function AlertInfo(props: Props) {
                     />
                     <TextOutput
                         label={strings.alertInfoWeb}
-                        value={data?.web}
+                        value={(
+                            <Link
+                                to={data?.web || ''}
```
Comment (samshara) [inline src/views/AlertDetails/AlertInfo/index.tsx]: Why empty string here?

File: src/views/AlertDetails/AlertMetadata/index.tsx [hunk 2]
```diff
@@ -133,7 +134,15 @@ function AlertMetadata(props: Props) {
             <MetaOutput
                 valueClassName={styles.url}
                 label="URL"
-                value={data?.url}
+                value={(
+                    <Link
+                        to={data?.url || ''}
```
Comment (samshara) [inline src/views/AlertDetails/AlertMetadata/index.tsx]: Let's not use empty string?

File: src/views/AlertDetails/index.tsx [hunk 3]
```diff
@@ -70,6 +70,11 @@ const GET_ALERT_DETAILS = gql`
                     language
                     id
                 }
+                admin1s {
+                    id
+                    filteredAlertCount
```
Comment (samshara) [inline src/views/AlertDetails/index.tsx]: any reason we fetch `filteredAlertCount` here? This is not used anywhere.

**Review by samshara — CHANGES_REQUESTED** [review]
Also, why there are additional commits here? As previously suggested, always create a branch from `develop` branch. Let's not forget this moving forward.

File: src/views/AlertDetails/AlertInfo/index.tsx [hunk 4]
```diff
@@ -232,7 +233,15 @@ function AlertInfo(props: Props) {
...
                         label={strings.alertInfoWeb}
-                        value={data?.web}
+                        value={data?.web && data.web !== ('') && (
+                            <Link
+                                to={data.web}
+                                className={styles.alertInfoWebLink}
+                                target="_blank"
+                            >
+                                {data.web}
+                            </Link>
```
Comment (barshathakuri) [inline src/views/AlertDetails/AlertInfo/index.tsx]: Use isTruthyString

**Review by barshathakuri — APPROVED** [review]

File: src/views/AlertDetails/AlertInfo/index.tsx [hunk 5]
```diff
@@ -3,6 +3,7 @@ import {
     useMemo,
     useState,
 } from 'react';
+import { Link } from 'react-router-dom';
```
Comment (samshara) [inline src/views/AlertDetails/AlertInfo/index.tsx]: Let's use Link from the components folder

File: src/views/AlertDetails/AlertMetadata/index.tsx [hunk 6]
```diff
@@ -1,3 +1,4 @@
+import { Link } from 'react-router-dom';
```
Comment (samshara) [inline src/views/AlertDetails/AlertMetadata/index.tsx]: Let's use Link from the components folder.

**Review by samshara — CHANGES_REQUESTED** [review]

File: src/views/AlertDetails/AlertInfo/index.tsx [hunk 7]
```diff
@@ -232,7 +233,16 @@ function AlertInfo(props: Props) {
                     />
                     <TextOutput
                         label={strings.alertInfoWeb}
-                        value={data?.web}
+                        value={isTruthyString(data?.web) && (
+                            <Link
+                                className={styles.alertInfoWebLink}
+                                href={data.web}
+                                target="_blank"
```
Comment (samshara) [inline src/views/AlertDetails/AlertInfo/index.tsx]: Let's remove the line as the Link component adds the target value internally.

File: src/views/AlertDetails/AlertMetadata/index.tsx [hunk 8]
```diff
@@ -132,7 +134,16 @@ function AlertMetadata(props: Props) {
             <MetaOutput
                 valueClassName={styles.url}
                 label={strings.alertMetaDataURL}
-                value={data?.url}
+                value={isTruthyString(data?.url) && (
+                    <Link
+                        className={styles.alertMetaDataUrl}
+                        href={data.url}
+                        target="_blank"
```
Comment (samshara) [inline src/views/AlertDetails/AlertMetadata/index.tsx]: Let's remove this line.

**Review by samshara — CHANGES_REQUESTED** [review]

**Review by samshara — APPROVED** [review]

**Review by barshathakuri — APPROVED** [review]

File: src/views/AlertDetails/AlertInfo/styles.module.css [hunk 9]
```diff
@@ -6,6 +6,16 @@
...
+    .alert-info-web-link{
+        text-decoration: none;
+        color: var(--go-ui-color-text);
+        font-weight: var(--go-ui-font-weight-medium);
+    }
+
+    .alert-info-web-link:hover {
+        text-decoration: underline;
+        color: var(--go-ui-color-primary-red);
+    }
```
Comment (samshara) [inline src/views/AlertDetails/AlertInfo/styles.module.css]: Now maybe we don't need these classses.

File: src/views/AlertDetails/AlertMetadata/styles.module.css [hunk 10]
```diff
@@ -21,7 +21,20 @@
...
+            .alert-meta-data-url{
+                text-decoration: none;
+                color: var(--go-ui-color-text);
+                font-weight: var(--go-ui-font-weight-medium);
+            }
+        
+            .alert-meta-data-url:hover {
+                text-decoration: underline;
+                color: var(--go-ui-color-primary-red);
+            }
```
Comment (samshara) [inline src/views/AlertDetails/AlertMetadata/styles.module.css]: Maybe we don't need these classes.

**Review by samshara — CHANGES_REQUESTED** [review]

**Review by barshathakuri — APPROVED** [review]

**Review by barshathakuri — APPROVED** [review]

**Review by samshara — APPROVED** [review]

**Pull request merged**